cmd/minikube/cmd/root.go

```diff
@@ -37,6 +40,11 @@ var (
 	showLibmachineLogs bool
 )
 
+//break this out into a package with config.UpdateNotification?
+const updateNotification = "updateNotification"
+
+var configFilePath = constants.Minipath + "/config"
```

Comment:
You can use MakeMinipath to concatenate htis correctly, and you can move this to constants.go since it'll probably be used in lots of places.